src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -40,7 +40,6 @@ public class CertificateGenerator
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
 
-        private const string _authorityCanonicalName = "DO_NOT_TRUST_WcfBridgeRootCA";
```

Comment:
Revert all changes in this file referring to Constants. `CertificateGenerator` is designed to have no dependencies on the Bridge or any test classes. 

If you wish, you may add a setter to the `AuthorityCanonicalName` and make sure all instances of `CertificateGenerator` initialize the `AuthorityCanonicalName` value to what's in Constants prior to calling Initialize().